atlassian_jwt_auth/contrib/flask_app/decorators.py

```diff
@@ -42,9 +42,13 @@ def decorated(*args, **kwargs):
 
 
 def _get_verifier():
-    """Returns a verifier based on config['ASAP_PUBLICKEY_REPOSITORY']"""
-    return atlassian_jwt_auth.JWTAuthVerifier(
-        atlassian_jwt_auth.HTTPSPublicKeyRetriever(
-            current_app.config['ASAP_PUBLICKEY_REPOSITORY']
-        )
+    """Returns a verifier for ASAP JWT tokens basd on application settings"""
+    retriever_cls = current_app.config.get(
```

Comment:
For such a short method I feel that the newlines do not help with readability at all and dislike them here.